src/style.css

```diff
@@ -0,0 +1,78 @@
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+  color: #fff;
+  --block-height: 300px;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  min-width: calc(var(--block-height * 3));
+}
+
+.block {
+  height: var(--block-height);
+  flex-basis: var(--block-height);
+  line-height: var(--block-height);
```

Comment:
To center block content vertically and horizontally you could use flex here as well, instead line 22 and 23 you could write 
`
    display: flex;
    align-items: center;
    justify-content: center;
`

Reply:
Yea, but for one liners content this makes one property less. Should i change that?